.github/workflows/rails.yml

```diff
@@ -22,16 +22,27 @@ jobs:
       DATABASE_URL: 'mysql2://127.0.0.1' # shogo82148/actions-setup-mysql@v1 doesn't work with socket
     steps:
       - uses: actions/checkout@v3
-#      - name: setup and start elasticsearch
-#        if: inputs.coyote-integration-tests
-#        uses: tedconf/elasticsearch-github-actions@v2.0.0
-#        with:
-#          stack-version: 6.8.23
-#          plugins: 'analysis-icu analysis-phonetic'
-#      - name: Elasticsearch is reachable
-#        if: inputs.coyote-integration-tests
-#        run: |
-#          curl --verbose --show-error http://localhost:9200
+      - name: Configure sysctl limits on localhost to be able to run Elasticsearch
+        # Required to get Elasticsearch container to run on shgar host
+        shell: bash
+        if: inputs.coyote-integration-tests
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
```

Comment:
It hasn't worked otherwise -- after some effort (undertaken by @ppagan-ted ), it seems that the Docker action's systctl configs are scoped to its own context, and isn't made available to the calling host, and so we have to specify it in this context.